app/views/landmarks/show.html.erb

```diff
@@ -1,9 +1,9 @@
 <h3><%= @landmark.name %></h3>
-<%= @landmark.user.email %>
+Creator: <%= @landmark.user.email %>
 <%= image_tag("#{@landmark.image}", height: '250', width: '250') %>
```

Comment:
@Salisbum I kind of indicated my opinion above about "when to style a feature" (when it's being built), but I will merge it if you have decided as a group that you want to do it later.